docs/source/algorithms/LoadISISNexus-v2.rst

```diff
@@ -14,13 +14,118 @@ Loads a Nexus file created from an ISIS instrument.
-Instrument information is loaded ``/raw_data_1/Instrument``, if available there and not overriden.
+The workspace data is loaded from ``raw_data_1/Detector_1``. 
+Instrument information is loaded ``raw_data_1/Instrument``, if available there and not overriden.
 Also the ``NSP1``, ``UDET``, ``SPEC``, ``HDR``, ``IRPB``, ``RRPB``, ``SPB`` and ``RSPB`` sections of
-``/raw_data_1/isis_vms_compat`` are read.
+``raw_data_1/isis_vms_compat`` are read.
+
+Here are some tables that show it in more detail:
+
++------------------------------+-------------------------------------------+-------------------------------------+
+| Description of Data          | Found in Nexus file                       | Placed in Workspace (Workspace2D)   |
+|                              | (within 'raw_data_1')                     |                                     |
++==============================+===========================================+=====================================+
+| Monitor Data                 | within groups of Class NXMonitor          | Depending on property LoadMonitors, | 
+|                              | (one for each monitor)                    | monitor histogram data              |
++------------------------------+-------------------------------------------+-------------------------------------+
+| Detector Data                | group ``Detector_1``                      | Histogram Data                      |
+|                              |                                           |                                     |
++------------------------------+-------------------------------------------+-------------------------------------+
+| Instrument                   | group ``Instrument``                      | Workspace instrument                |
+|                              |                                           | if not overridden                   |
++------------------------------+-------------------------------------------+-------------------------------------+ 
+| Spectrum of each detector ID | ``NSP1``, ``UDET`` and ``SPEC``           | Spectra-Detector mapping            |
+|                              | within ``isis_vms_compat``                |                                     |
++------------------------------+-------------------------------------------+-------------------------------------+  
+| Run                          | various places as shown later on          | Run object                          |
++------------------------------+-------------------------------------------+-------------------------------------+
+| Sample                       | ``SPB`` and ``RSPB`` within               | Sample Object                       | 
+|                              | ``isis_vms_compat``                       |                                     |
++------------------------------+-------------------------------------------+-------------------------------------+
+
```

Comment:
Does it load the run logs? That normally includes important logs like `total_counts` or `good_frames`. I'd expect to see the logs in this table or some other table(s).